### PR TITLE
pybess: fix nested protobuf dict parsing

### PR DESCRIPTION
**Pull request by melvinw**

### CODE REVIEW

File: pybess/protobuf_to_dict.py [hunk 1]
```diff
@@ -154,8 +154,14 @@ def _dict_to_protobuf(pb, value, type_callable_map, strict):
 
     for field, input_value, pb_value in fields:
         if field.label == FieldDescriptor.LABEL_REPEATED:
-            if field.message_type and field.message_type.has_options and field.message_type.GetOptions().map_entry:
-                pb_value.update(input_value)
+            if (field.message_type and field.message_type.has_options and
```
Comment (changlan) [inline pybess/protobuf_to_dict.py]: nit: remove parenthesis?

Comment (melvinw) [inline pybess/protobuf_to_dict.py]: Fixed

Comment (changlan) [inline pybess/protobuf_to_dict.py]: Ah, a `\` at the end of line is missing.

Comment (chris3torek) [inline pybess/protobuf_to_dict.py]: Ah, yes, you need parens, `\`-at-end-of-line, or squash-it-all-into-one-line. Personally I prefer the parentheses 😄 but I suppose it is "sixes" as some say.

Comment (melvinw) [inline pybess/protobuf_to_dict.py]: Ah, whoops! Fixed

**Comment by changlan** [comment]
image

**Pull request merged**